.github/workflows/test.yml

```diff
@@ -0,0 +1,78 @@
+name: Build and Test
+
+on:
+  pull_request:
+    branches:
+      - 'test/*'
+
+env:
+  ARTIFACTORY_URL: ${{ secrets.ARTIFACTORY_URL }}
```

Comment:
I don't see this secret in the repo. Won't this test fail as a result?

Reply:
We just got the artifactory creds and everything going to add that in today.

Reply:
@oluwolenpbc  Seems to me like you could create another branch off of this one, name it with the 'test/...' prefix and change to what @janejr11 proposed to test (on: push: branches: test/oluwolenpbc-patch-1) and be able to test from there.